apps/create-vesting/src/app/page.tsx

```diff
@@ -78,6 +80,11 @@ export default function Page() {
 
   if (context?.hookToEdit && isEditHookLoading) {
     loadHookInfo();
+    return (
+      <div className="flex items-center justify-center w-full h-full bg-transparent text-color-text-paper">
+        <Spinner size="lg" style={{ color: "gray" }} />
```

Comment:
Could you explain why you chose to use "gray" spinner, I think that we should use the same that is being used on the other apps.